src/features/definitionProvider.ts

```diff
@@ -60,6 +60,9 @@ export default class CSharpDefinitionProvider extends AbstractSupport implements
                     return new Location(uri, new Position(gotoDefinitionResponse.Line - 1, gotoDefinitionResponse.Column - 1));
                 });
             }
-        });
+        }
+        catch (error) {
+            return;
```

Comment:
I'm not going to make the same comment again but can you go through and make sure we return something (even if null in non-void methods)?